### PR TITLE
Feature/prometheus

### PR DESCRIPTION
**Pull request by tomverelst**

### CODE REVIEW

**Comment by tomverelst** [comment]
Please review. **Do not merge yet**.

File: _posts/2016-09-23-Monitoring-with-Prometheus.md [hunk 1]
```diff
@@ -0,0 +1,667 @@
...
+title: 'Monitoring with Prometheus'
+image: /img/prometheus.jpg
+tags: [Prometheus]
+category: Monitoring
+comments: true
+---
+
+It is needless to say the world is shifting towards DevOps and microservices.
+This holy grail we all aim for adds a lot of complexity,
+as well for monitoring.
```
Comment (tomvdbulck) [inline _posts/2016-09-23-Monitoring-with-Prometheus.md]: ligt niet zo lekker in de mond: ofwel "especially for monitoring" of "for the general architecture as well as for monitoring"

File: _posts/2016-09-23-Monitoring-with-Prometheus.md [hunk 2]
```diff
@@ -0,0 +1,667 @@
...
+or monitoring the CPU usage.
+In a world of distributed architectures where high availability and fast response times are key,
+it is not sufficient to be only aware if a service is alive.
+It is crucial to know how a service is working internally as well.
+How many HTTP requests is it receiving?
+Are they handled correctly?
+How fast are requests handled for different endpoints?
+Are there many errors being logged?
+How many disk IO operations is the service making?
+These are all important questions that need to be asked to keep a service functional.
```
Comment (tomvdbulck) [inline _posts/2016-09-23-Monitoring-with-Prometheus.md]: that need to be asked => that need to be monitored in order to keep a service functional.

File: _posts/2016-09-23-Monitoring-with-Prometheus.md [hunk 3]
```diff
@@ -0,0 +1,667 @@
...
+Combining all the above tools,
+we can get the rates of HTTP requests of a specific timeframe.
+The query below will calculate the per-second rates of all HTTP requests
+that occurred in the last 5 minutes an hour ago:
+
+```javascript
+rate(api_http_requests_total{status=500}[5m] offset 1h)
+```
+
+A slightly more complex example selects the top 3 endpoints which have the most HTTP requests
```
Comment (tomvdbulck) [inline _posts/2016-09-23-Monitoring-with-Prometheus.md]: selects the top 3 endpoints which => for which

File: _posts/2016-09-23-Monitoring-with-Prometheus.md [hunk 4]
```diff
@@ -0,0 +1,667 @@
...
+[Ruby](https://github.com/prometheus/client_ruby).
+There is also a lengthy list of unofficial third-party clients for other languages,
+including clients for Bash and Node.js.
+These clients enable you to expose metrics endpoints through HTTP.
+
+This is totally different compared to other,
+more traditional,
+monitoring tools.
+Normally,
+the application is unaware that is being monitored.
```
Comment (tomvdbulck) [inline _posts/2016-09-23-Monitoring-with-Prometheus.md]: that is => that it is

File: _posts/2016-09-23-Monitoring-with-Prometheus.md [hunk 5]
```diff
@@ -0,0 +1,667 @@
...
+* [Architecture](#architecture)
+* [Data Model](#data-model)
+* [Slice &amp; Dice with the Query Language](#slice-dice-with-the-query-language)
+* [Instrumenting Your Services](#instrumenting-your-services)
+* [Exporters](#exporters)
+* [Scraping the Targets](#scraping-the-targets)
+* [Visualization and Analytics](#visualization-and-analytics)
+* [Alert! Alert!](#alert-alert)
+* [Monitoring Time!](#monitoring-time)
+
```
Comment (tomvdbulck) [inline _posts/2016-09-23-Monitoring-with-Prometheus.md]: Geen link naar Final Words

File: _posts/2016-09-23-Monitoring-with-Prometheus.md (hunk 3 above)
Comment (tomverelst) [inline _posts/2016-09-23-Monitoring-with-Prometheus.md]: "which" is correct here

**Pull request merged**